probe-rs/src/vendor/nordicsemi/sequences/nrf91.rs

```diff
@@ -23,26 +21,24 @@ impl Nrf9160 {
 impl Nrf for Nrf9160 {
     fn core_aps(
         &self,
-        memory: &mut dyn ArmMemoryInterface,
+        ap_address: &FullyQualifiedApAddress,
```

Comment:
It seems that you don't actually use this, but only get the DP address from it. I'd prefer to just pass the `DpAddress` here.

Reply:
Done!